features/lorawan/lorastack/mac/LoRaMac.cpp

```diff
@@ -889,6 +889,8 @@ void LoRaMac::on_backoff_timer_expiry(void)
 
 void LoRaMac::open_rx1_window(void)
 {
+    tr_debug("Opening RX1 Window");
```

Comment:
No. Opening RX windows is quite time critical so let's not print until the window has been opened.